docs/advanced-features/workflows.md

```diff
@@ -623,6 +629,31 @@ Requires a README
 }
 ```
 
+##### `s3://bucket/must-have-foo-bar-meta.json`
+
+Requires `{ foo: 'bar' }` object as user specified metadata
+
+```json
+{
+  "type": "array",
+  "contains": {
+    "type": "object",
+    "properties": {
+      "meta": {
+        "type": "object",
+        "properties": {
+          "foo": {
+            "type": "string",
+            "pattern": "^bar$"
+          }
+        },
+        "required": ["foo"]
```

Comment:
Just in case you want it to be exactly `{ "foo": "bar" }`, you'd need to set `"additionalProperties": false` or just use `const`.

Reply:
Got it, but my intention was to have any fields AND require `{ "foo": "bar" }`